### PR TITLE
Add tic tac toe game implementation

### LINKED ISSUE #2: Team-99 code reivew
- Nice work done. But what I'll suggest is that there should be a variable that is assigned the board's cells so that after the game has ended you can use it reset the board.

### PR DESCRIPTION
**Pull request by Bluette1**
## What does this PR do?
- [x] It adds the tic tac toe game logic and User interface
- [x] It fixes [#2](https://github.com/Bluette1/tic-tac-toe/issues/2)

## Tasks to be completed on this PR
- [x] Design OOP structure
- [x] Add front-end display
- [x] Use factories
- [x] ES6 syntax

## Deployed site
[Visit link](https://eloquent-hopper-9cff17.netlify.app/)

### CODE REVIEW

File: javascript/gameFlow.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        counter += 1;
+      }
+    }
+    if (counter >= 9) {
+      return true;
+    }
+    return false;
+  };
+  const boardReset = () => [['', '', ''], ['', '', ''], ['', '', '']];
+  // const resetGame = (gameObj) => {};
```
Comment (uche-inyama) [inline javascript/gameFlow.js]: There is a commented line of code, do remove it.

**Review by uche-inyama — APPROVED** [review]
## Approved

### Hi team :wave: 

Good job. 

However, don't forget to remove the commented line of code. I have highlighted it below.

[**OPTIONAL**] It would be nice, to rename **```JavaScript```**  folder to **```src```** folder

#### ```Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry Inyama ```

File: javascript/gameFlow.js (hunk 1 above)
Comment (descholar-ceo) [reply javascript/gameFlow.js]: We have just implemented this, thank you!

**Comment by descholar-ceo** [comment]
> ## Approved
> ### Hi team 
> Good job.
> 
> However, don't forget to remove the commented line of code. I have highlighted it below.
> 
> [**OPTIONAL**] It would be nice, to rename **`JavaScript`** folder to **`src`** folder
> 
> #### `Have a great day ` 
> `Uchechukwu Henry Inyama`

@uche-inyama Thank you so much for this kind review, we have implemented all of the optional requested changes and we appreciate this contribution of yours which made us improve our work.

**Pull request merged**